sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BeamRowToStorageApiProtoTest.java

```diff
@@ -392,4 +415,11 @@ public void testMessageFromTableRow() throws Exception {
     DynamicMessage nestedMsg = (DynamicMessage) msg.getField(fieldDescriptors.get("nested"));
     assertBaseRecord(nestedMsg);
   }
+
+  @Test
```

Comment:
Thanks for the finding. However this test looks irrelevant to the class we test against. We could simply replace "sqlTimestampValue" with a constant timestamp with micros precision, like those "int64Value", "byteValue" in BASE_ROW

Reply:
Ahh yes, I wrote that test just to confirm that the system UTC clock on the Jenkins machine was in fact using millis precision.

I logged the output of `Instant.now()` (see [here](https://ci-beam.apache.org/job/beam_PreCommit_Java_GCP_IO_Direct_Commit/1193/testReport/org.apache.beam.sdk.io.gcp.bigquery/BeamRowToStorageApiProtoTest/testInstant/)) and got `2022-12-27T19:32:26.036Z`

Reply:
will remove the test now!

Reply:
+1 to extending sqlTimestampValue micros precision